src/audio/mixin_mixout/mixin_mixout.c

```diff
@@ -115,12 +114,6 @@ static int mixin_init(struct processing_module *mod)
 		md->sink_config[i].gain = IPC4_MIXIN_UNITY_GAIN;
 	}
 
-	audio_stream_fmt_conversion(mod->priv.cfg.base_cfg.audio_fmt.depth,
```

Comment:
@btian1 you're removing the assignment of the frame_fmt in the ipc_config but the commit message talks about something else totally. Can you please fix it?